src/utils.ts

```diff
@@ -602,6 +602,27 @@ export const throttle = <T extends (...args: unknown[]) => unknown>(
   };
 };
 
+const get = <T>(obj: T, path: string): unknown =>
+  path.split('.').reduce<unknown>((acc, key) => {
+    if (acc && typeof acc === 'object' && key in acc) {
+      return (acc as Record<string, unknown>)[key];
+    }
+    return undefined;
+  }, obj);
+
+// works exactly the same as lodash.uniqBy
+export const uniqBy = <T>(array: T[] | unknown, iteratee: ((item: T) => unknown) | keyof T): T[] => {
```

Comment:
Not the same implementation as in `lodash` obviously and it does not use their fast `SetCache`, but close enough for such small lists. Still ~ `O(n)`.